app/code/Magento/Sitemap/Api/Data/SitemapInterface.php

```diff
@@ -0,0 +1,116 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Sitemap\Api\Data;
+
+/**
+ * @SuppressWarnings(PHPMD.CouplingBetweenObjects)
+ */
+interface SitemapInterface
```

Comment:
It would make sense to add support for extension attributes to this interface.

Reply:
Created SitemapExtensionInterface + all related things and added them